chapter04_convolutional-neural-networks/kaggle-gluon-cifar10.md

```diff
@@ -0,0 +1,412 @@
+# 实战Kaggle比赛——使用Gluon对原始图像文件分类（CIFAR-10）
+
+
+我们在[监督学习中的一章](../chapter02_supervised-learning/kaggle-gluon-kfold.md)里，以[房价预测问题](https://www.kaggle.com/c/house-prices-advanced-regression-techniques)为例，介绍了如何使用``Gluon``来实战[Kaggle比赛](https://www.kaggle.com)。社区中的很多小伙伴实践了房
+价预测问题并[分享了自己的成绩和方法](https://discuss.gluon.ai/t/topic/1039)。有小伙伴[反馈希望增加Kaggle比赛章节](https://discuss.gluon.ai/t/topic/1341)并[反馈希望多从直接与数据打交道中获得灵感](https://discuss.gluon.ai/t/topic/1519)。还有小伙伴[反馈希望提供对原始图像分类的例子](https://discuss.gluon.ai/t/topic/499)，
+例如输入格式是jpg或png而非教程中封装好的图像数据，这样甚至可能[更好地管理手机图片](https://discuss.gluon.ai/t/topic/1372)。
+
+
+有鉴于大家的反馈，我们在本章中选择了Kaggle中著名的[CIFAR-10原始图像分类问题](https://www.kaggle.com/c/cifar-10)。我们以该问题为例，为大家提供使用`Gluon`对原始图像文件进行分类的示例代码。
+
+本章的示例代码主要分五部分：
+
+* 整理原始数据集；
+* 使用`Gluon`读取整理后的数据集；
+* 设计模型；
+* 使用部分训练数据训练模型并以其它训练数据为验证集调参；
+* 使用全部训练数据训练模型并对测试数据分类。
+
+
+需要注意的是，本章仅提供一些基本实战流程供大家参考。对于数据的预处理、模型的设计和参数的选择等，我们特意提供一个仅供示例的版本。希望大家通过动手实战、仔细观察实
```

Comment:
we can move it into the end, so that we can present real staff earlier.

Reply:
resolved